src/ui/index.js

```diff
@@ -8,6 +8,13 @@ import UiExports from './ui_exports';
 import UiBundle from './ui_bundle';
 import UiBundleCollection from './ui_bundle_collection';
 import UiBundlerEnv from './ui_bundler_env';
+import i18nPlugin from '../core_plugins/i18n/server/i18n/index';
+import langParser from 'accept-language-parser';
+
+let kibanaTranslations = [];
+let acceptLanguages = '';
+
+const DEFAULT_LOCALE = 'en';
```

Comment:
If you add this as a default in the config schema it could be configured in kibana.yml: https://github.com/Bargs/kibana/blob/ea57d861887797de218a3a5943926d92e7244406/src/server/config/schema.js